config/dbSetup.sh

```diff
@@ -47,6 +47,9 @@ if [ "$OPEN_WHISK_DB_PROVIDER" == "Cloudant" ]; then
         echo "Cloudant password is not set"
         exit 1
     fi
+
```

Comment:
That was the case in a previous version, but it broke a deploy script. My theory is that `CLOUDANT_USERNAME` is set multiple times to different values during the build, and `OPEN_WHISK_DB_USERNAME` captures it at the wrong time (for that particular build). It did work for local builds though.